File: 백준/Bronze/34722. 출제자가 몇 명/출제자가 몇 명.cc
#include <iostream>
using namespace std;
int main(void) {
    int n,si,ci,ai,ri,cnt=0;
    cin>>n;
    for(int i=0;i<n;i++) {
        cin>>si>>ci>>ai>>ri;
        if(si>=1000 || ci>=1600 || ai>=1500 || (1<=ri&&ri<=30))
            cnt+=1;
    }
    cout<<cnt;
    return 0;
}

File: 백준/Bronze/34722. 출제자가 몇 명/출제자가 몇 명.py
import sys
input=lambda:sys.stdin.readline().rstrip()
cnt=0
for i in range(int(input())):
    si,ci,ai,ri=map(int,input().split())
    if si>=1000 or ci>=1600 or ai>=1500 or 1<=ri<=30:
        cnt+=1
print(cnt)